Project_ProposalTemplate_AWaggoner.md

```diff
@@ -0,0 +1,121 @@
+
+    How did these changes impact the Black communities living within Brooklyn? Initially, during the late 1800s, as freed slaves began to escape and live towards the north, Black populations rose in many urban areas like Philadelphia, Washington, D.C., and other cities across the United States. New York was another city where freed Blacks lived and escaped towards, and similar to Harlem, Brooklyn was another location within New York City where Blacks felt safe to establish community in. Because of the community that Blacks established in Brooklyn, many types of businesses: (churches, restaurants, schools, nightlife, etc.), were established for and by Black Brooklynites. 
+
+As time progressed, closer to the new millennium, white gentrifiers
+replaced and outnumbered the Black populace of Brooklyn. Many formerly
+majority Black neighborhoods within Brooklyn were becoming increasingly
+more populated by “upper-class whites” (Chronopoulos 2020). Because of
+these changes, many Black Brooklynites felt displaced and removed from
+the communities they established and created, feeling as if their
+legacies were being removed from the neighborhoods they inhabited.
+Gentrification not only displaced Black inhabitants from their homes but
+also displaced Black Brooklynites from their businesses and general
+sense of community (Valli 2016).
+
+# Problem / Question
+
+This project will analyze gentrification in Brooklyn, NYC via Black
+music venues. Using R and spatial data, I will answer the following
+research questions: How Black music venues-as an extension of the
+greater, Black community within Brooklyn-were affected as gentrification
+ensued over the new millennium? What are the ways in which Black music
+venues were erased from their neighborhoods within Brooklyn? What venues
+still remain? How has the Black population shifted in accordance with
+the changes in these venues over time?
+
+# Inspiring Examples
+
+Below are a series of examples that describe similar findings using
+demographic data and maps.
+
+## Example 1
+
+![](http://www.zevross.com/blog/wp-content/uploads/2018/09/map_facet_long.png)
+
+This graphic displays a demographic data mapped over time. Although,
+this is not business data, this is a similar style of map that I want to
+create. \#\# Example 2
+
+![](http://www.zevross.com/blog/wp-content/uploads/2018/09/map_facet_wide.jpg)
+Another example of the mapping style that I want to have - comparing
+demographic data that has changed over time.
+
+# Proposed Data Sources
+
+Data sources will contain:
+
+1.  Census Tract shapefiles of racial demographics from 2000 to 2020.
+    (The goal for the purposes of this class project is to breakdown the
+    racial data by every ten years so 2000, 2010, 2020, respectively.)
+
+2.  Yelp API (This will be used to download the information from
+    Black-owned music venues from 2000 to 2020. Similar to the census

```

Comment:
does this api have information on ownership?  how will you know which are black-owned?